### ISSUE TITLE
R Package Lab - Masking Error

### ISSUE THREAD

**Issue by CTNovoa** [Watts-College/paf-514-template#74]
I am getting this masking error when I attempt devtools::document()

![image](https://github.com/user-attachments/assets/2e58ee3a-3c1e-44be-be62-e6b603dda33f)

Any ideas as to why this might be coming up and how to resolve this? Thank you!

**Comment by CTNovoa** [comment]
Follow-up: when I try loading a game in RGui it does seem to be working properly, so I am not sure how concerned I should be about this "objects are masked _by_ '.GlobalEnv' error"

**Comment by castower** [comment]
Hello @CTNovoa ,
What is your working directory when you run devtools::document()? Sometimes this message will occur if you aren't in the main montyhall folder and a subfolder instead.

**Comment by CTNovoa** [comment]
Thanks for that assistance, @castower! I had checked that and was in the main montyhall folder. 

Just an update that I uploaded this most recent attempt on this lab to GitHub and tested it and it all seems to be functioning. So I am not sure if that means that this masking message is not an issue? 

This is also my second attempt at this lab, as I wanted to solidify my understanding (the second run-through was so helpful to me--thanks for the extra time to do so!) and so maybe it is picking up on other file names (etc.) that are already saved to my computer from my earlier attempt at this lab? I could be misunderstanding what this "masking" means, but I just wanted to share that in case that is helpful information.
